invenio_app_rdm/config.py

```diff
@@ -657,6 +657,12 @@ def files_rest_permission_factory(obj, action):
         "content-type": "application/vnd.geojson+json",
         "filename": "{id}.geojson",
     },
+    "DCAT-AP": {
+        "name": _("DCAT"),
+        "serializer": "invenio_rdm_records.resources.serializers:DCATSerializer",
+        "content-type": "text/turtle",
+        "filename": "{id}.ttl",
```

Comment:
```suggestion
        "content-type": "application/dcat+xml",
        "filename": "{id}.xml",
```

Based on https://github.com/inveniosoftware/invenio-rdm-records/pull/1233#discussion_r1134259462